### ISSUE TITLE
incorrect binding direction transfer

### ISSUE THREAD

**Issue by KhaledSMQ** [NG-ZORRO/ng-zorro-antd#4795]
### Reproduction link
[https://stackblitz.com/edit/ng-zorro-antd-start-u212hy?file=src%2Fapp%2Fapp.component.ts](https://stackblitz.com/edit/ng-zorro-antd-start-u212hy?file=src%2Fapp%2Fapp.component.ts)

### Steps to reproduce
the model binding must be 

```typescript
[direction]="'left'"
```

[left](https://github.com/NG-ZORRO/ng-zorro-antd/blob/7c055117d8a83fa83f05a92e28b5861427e9cd4c/components/transfer/nz-transfer.component.html#L4)

the model binding must be 
```typescript
[direction]="'right'"
```

[right](https://github.com/NG-ZORRO/ng-zorro-antd/blob/7c055117d8a83fa83f05a92e28b5861427e9cd4c/components/transfer/nz-transfer.component.html#L47)

### What is expected?
correct the model binding, I am not sure where the direction setter in nz-transfer-list using angular ^8, but in angular 9 it's not working.

### What is actually happening?
let-direction="direction" receiving empty string in angular 9

| Environment | Info |
|---|---|
| ng-zorro-antd | 8.5.2 |
| Browser | all |

---
![correct binding](https://i.ibb.co/jy2QFcP/correct-bind.png)

![incorrect binding](https://i.ibb.co/4SSYmrp/incorrect-bind.png)

**Comment by KhaledSMQ** [comment]
@Ricbet  @vthinkxie , Hi sorry this bug not solved even with the new release, 
you can verify even on the NgZorro website.

https://ng.ant.design/components/transfer/en#components-transfer-demo-table-transfer

in the example you are checking the direction 
```
              <th *ngIf="direction === 'left'">Tag</th>
.
.
.
             <td *ngIf="direction === 'left'">
                <nz-tag>{{ data.tag }}</nz-tag>
              </td>

```
but the value is not passed to the template so it will never show the tag as per example.

**Comment by KhaledSMQ** [comment]
solved with new release 9.1.0
